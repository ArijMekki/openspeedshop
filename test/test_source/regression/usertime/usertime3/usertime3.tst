echo expcreate -f ../../../../executables/mutatee/f77_version/mutatee usertime >> input.script
echo expgo >> input.script
echo expview stats -m usertime::inclusive_time >>input.script
openss -batch < input.script
