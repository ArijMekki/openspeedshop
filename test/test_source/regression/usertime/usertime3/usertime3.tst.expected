      Inclusive Time  Function Name
              6.0000  __libc_start_main(libc.so.6.1)
              6.0000  main(mutatee)
              6.0000  _start(mutatee)
              6.0000  MAIN_(mutatee: mutatee.f,1)
              6.0000  work(mutatee: mutatee.f,13)
              3.0000  f3(mutatee: mutatee.f,25)
              2.0000  f2(mutatee: mutatee.f,34)
              1.0000  f1(mutatee: mutatee.f,44)
